ADAM/src/AppBundle/Resources/views/advert/index.html.twig

```diff
@@ -4,11 +4,14 @@
 <section class="pageAdvertList">
     <h1>Listes des annonces</h1>
 
-    <div align="right">
```

Comment:
Je n'ai pas testé mais... pourquoi avoir bougé ce bloc dans la boucle `for` ? Ça veut dire que le bouton de création d'annonce va être placé à côté de chaque annonce ?